librz/analysis/p/analysis_arm_cs.c

```diff
@@ -991,8 +991,43 @@ static void anop64(ArmCSContext *ctx, RzAnalysisOp *op, cs_insn *insn) {
 	case CS_AARCH64(_INS_CMN):
 	case CS_AARCH64(_INS_TST):
 #endif
+		if (ISIMM64(1)) {
+			op->val = IMM64(1);
+		}
 		op->type = RZ_ANALYSIS_OP_TYPE_CMP;
 		break;
+#if CS_NEXT_VERSION >= 6
+	case CS_AARCH64(_INS_ADDS):
+		if (insn->alias_id == AArch64_INS_ALIAS_CMN) {
```

Comment:
Maybe it is worth to add an `inline` function which does:

```c
static inline bool is_alias(cs_insn insn, AArch64Alias alias_id) {
   return insn->is_alias && (insn->alias_id == alias_id);
}
```

Simply because this pattern gets more and more common.

Reply:
definetely